update.go

```diff
@@ -79,14 +79,35 @@ func (m *Manager) InstallTo(path, dir string) error {
 
 	dst := filepath.Join(dir, m.Command)
 
-	log.Debugf("move %q to %q", bin, dst)
-	if err := os.Rename(bin, dst); err != nil {
-		return errors.Wrap(err, "moving")
+	log.Debugf("copy %q to %q", bin, dst)
+
+	if err := copy(bin, dst); err != nil {
+		return errors.Wrap(err, "copy")
 	}
 
 	return nil
 }
 
+func copy(src, dst string) error {
```

Comment:
Maybe rename to `copyFile` or something since `copy()` is a built-in. Also needs a comment :D